src/object-properties.js

```diff
@@ -140,11 +142,13 @@ const properties = {
  * @property {PaletteColor=} color - Color defined by index or hex code, needed if useColorExpression is false
  * @property {ColorExpression=} colorExpression - Color defined by expression, needed if useColorExpression is true
  * @property {number=} [size=0.5] - Relative[] size of label, must be greater than 0 and 1 fills the entire button
+ * @property {number=} [sizeFixed=20] - Fixed size of the label in pixels. Must be bigger than 5. Only active when sizeBehavior is fixed.
```

Comment:
@haxxmaxx The api specifications would be updated when actually releasing right?

Reply:
that is the way we do it for the table, guessing same here. The API governance will complain if the api specs don't match with the specs for the current version I think